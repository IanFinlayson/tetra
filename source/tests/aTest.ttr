#Tests array assignment

def main():
   array1 = [1,2,3,4,5]
   print("Just created array1 and array1 = ", array1, "\n")
   array2 = [6 ... 10]
   print("Just created array2 and array1 = ", array2, "\n")
   array1 = array2
   print("Just changed set array1 = array2 and array1 = ", array1, "\n")

   sum = 0
   print("Just created sum and sum = ", sum, "\n")
   for num in array1:
      print("In for loop and num = ", num, "\n")
      sum += num
      print("Just added num to sum and sum = ", sum, "\n")
