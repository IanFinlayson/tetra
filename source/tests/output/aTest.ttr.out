Just created array1 and array1 = [1, 2, 3, 4, 5]
Just created array2 and array1 = [6, 7, 8, 9, 10]
Just changed set array1 = array2 and array1 = [6, 7, 8, 9, 10]
Just created sum and sum = 0
In for loop and num = 6
Just added num to sum and sum = 6
In for loop and num = 7
Just added num to sum and sum = 13
In for loop and num = 8
Just added num to sum and sum = 21
In for loop and num = 9
Just added num to sum and sum = 30
In for loop and num = 10
Just added num to sum and sum = 40

